editor/src/components/canvas/canvas-strategies/strategies/grid-helpers.ts

```diff
@@ -191,25 +167,138 @@ export function runGridRearrangeMove(
         updateGridControlsCommand,
       ]
     }
-    case 'reorder': {
-      return [
-        reorderElement('always', pathForCommands, absolute(possiblyReorderIndex)),
-        deleteProperties('always', pathForCommands, [
-          PP.create('style', 'gridColumn'),
-          PP.create('style', 'gridRow'),
-          PP.create('style', 'gridColumnStart'),
-          PP.create('style', 'gridColumnEnd'),
-          PP.create('style', 'gridRowStart'),
-          PP.create('style', 'gridRowEnd'),
-        ]),
-        updateGridControlsCommand,
-      ]
-    }
     default:
       assertNever(moveType)
   }
 }
 
+export function runGridMoveReorder(
```

Comment:
this is just reorder, right? no need to have move in the name anymore

Reply:
I agree but for consistency I kept the naming prefixes, imho naming must be fixed (e.g. "rearrange") but incrementally